scripts/requirements-fixed.txt

```diff
@@ -2,361 +2,132 @@
 # This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile --allow-unsafe --output-file=requirements-fixed.txt --strip-extras ./bootloader/mcuboot/scripts/requirements.txt ./nrf/scripts/requirements-ci.txt ./nrf/scripts/requirements-extra.txt ./nrf/scripts/requirements.txt ./zephyr/scripts/requirements.txt
+#    pip-compile --allow-unsafe --annotation-style=line --output-file=/home/runner/work/sdk-nrf-testing/sdk-nrf-testing/ncs/nrf/scripts/requirements-fixed.txt --strip-extras ./bootloader/mcuboot/scripts/requirements.txt ./nrf/scripts/requirements-ci.txt ./nrf/scripts/requirements-extra.txt ./nrf/scripts/requirements.txt ./zephyr/scripts/requirements.txt
```

Comment:
can we avoid invoking `pip-compile` with the full path to the out file ?
Ref: `--output-file=/home/runner/work/sdk-nrf-testing/sdk-nrf-testing/ncs/nrf/scripts/requirements-fixed.txt`

Especially because it's unfortunate to simply have changes to a file simply because a github workflow uses a different path.

Reply:
I don't think the path will change in the future.
This change is only highlighted here as I used a locally generated file to trigger the pipeline.
Will check if it is possible to drop the full path and only use relative format

Reply:
if it's ok with Torsten, I propose we merge this as is (since the path is generated in a file that isn't in this PR) 

and then look at printing a relative path when we 'backport/upmerge' this to sdk-nrf-next.